sdk/core/ci.yml

```diff
@@ -32,6 +32,7 @@ stages:
     parameters:
       ServiceDirectory: core
       CtestRegex: azure-core.
+      # no live tests, all tests are currently for CI only
       LiveTestCtestRegex: azure-core.
```

Comment:
Did you intend to remove this line? If not, how does the comment hold true?

Reply:
It should be commented. 
I uncommented in the last commit to test that even if live test is enabled we re-run the CI tests, there is no issues